content/kb/deployments/organize-apps-workspaces.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Organizing your apps with workspaces on Streamlit Cloud
+slug: /knowledge-base/deploy/organizing-apps-workspaces-streamlit-cloud
+---
+
+# Organizing your apps with workspaces on Streamlit Cloud
+
+Streamlit Cloud is organized into workspaces, which automatically group your apps according to the corresponding GitHub repository's owner. If you are part of multiple repositories, then you will have multiple workspaces.
+
+![App workspaces 1](/images/streamlit-cloud/app-workspaces-1.gif)
+
+## Personal workspace
+
+If an app's GitHub repository is owned by you, the app will appear in your personal workspace, named "<YourGitHubHandle\>".
+
+![App workspaces 2](/images/streamlit-cloud/app-workspaces-2.png)
+
+## Organization workspace
+
+If an app's GitHub repository is owned by **an organization** (such as your company), the app will appear in a separate workspace, named "<GitHubOrganizationHandle\>".
```

Comment:
There's an errant space in the quotation mark around githuborghandle, which causes a weird line break

Reply:
@randyzwitch I neither see the errant space nor the weird line break :thinking: . Would you mind sharing a screenshot of the line break?

Reply:
I see the line break in the GitHub UI, but not when inspecting the file content locally or on the rendered page. Not sure how to proceed :sweat_smile:

Reply:
That is so weird, it's gone now...that's an odd thing for me to have imagined, yet, no commits have made between now and then.

Ready for merge :exploding_head: